.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
Ah yeah my bad
1. Set the language of the project to node_js
2. Set the current version of the nodejs instance running on the travis testing container to the latest version of the 5.x branch of nodejs.  We can set specific versions if we need to, but for now I think latest should be alright.
3.  When we do the install step, run `npm install` to install dependencies.  Commands in the install section are always run before commands in the script section, so we will have our dependencies installed for our test
4. Run tests using the mocha test runner.  Set the timeout of the tests to 5000 milliseconds (for the integration tests).  The default time is 2000 ms, which we could potentially hit with a valid request.